src/index.html

```diff
@@ -7,8 +7,50 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header_block">
+      <div class="container">
+        <a href="index.html" class="logo">
+          <img src="images/logo.png" alt="logo MOYO">
+        </a>
+        <nav class="menu">
+          <ul class="menu_list">
+            <li>
+              <a href="" class="menu_link is-active">Apple</a>
+            </li>
+            <li>
+              <a href="" class="menu_link">Samsung</a>
+            </li>
+            <li>
+              <a href="" class="menu_link">Смартфоны и телефоны</a>
+            </li>
+            <li>
+              <a href=""
+                class="menu_link"
+                data-qa="hover"
+              >
+               Ноутбуки и компьютеры
+              </a>
+            </li>
+            <li>
+              <a href="" class="menu_link">Гаджеты</a>
+            </li>
+            <li>
+              <a href="" class="menu_link">Планшеты</a>
+            </li>
+            <li>
+              <a href="" class="menu_link">Фото</a>
+            </li>
+            <li>
+              <a href="" class="menu_link">Видео</a>
```

Comment:
```suggestion
              <a href="#" class="menu_link">Видео</a>
```
Не оставляй атрибут href пустым